README.md

```diff
@@ -73,14 +73,29 @@ If you have Apache Thrift installed you can regenerate these classes by using th
 
 # Execution
 
+## Set the PYTHONPATH variable
+
+    Mac:
+    
+        export PYTHONPATH=/path/to/dragonchain
+        
+    Linux:
+    
+        PYTHONPATH="/path/to/dragonchain"
+
 ## Transaction Service
 
     python <Dragonchain Home>/blockchain/transaction_svc.py --private-key <Dragonchain Home>/pki/sk.pem --public-key <Dragonchain Home>/pki/pk.pem
     
 ## Query Service
 
-    python <Dragonchain Home>/blockchain/query_svc.py --private-key <Dragonchain Home>/pki/sk.pem --public-key <Dragonchain Home>/pki/pk.pem
+    python <Dragonchain Home>/blockchain/query_svc.py -p port //defaults to 8080
```

Comment:
Let's try making these look more like standard CLI instructions. See Argparse for example (https://docs.python.org/3/library/argparse.html).

I'm thinking something like `query_svc.py [-p port]` would be more appropriate.